src/cli.rs

```diff
@@ -186,14 +188,16 @@ pub struct Img {
     ///
     ///Possible transitions are:
     ///
-    ///simple | left | right | top | bottom | center | outer | any | random
+    ///simple | left | right | top | bottom | center | outer | any | random | wipe
```

Comment:
This is an incredibly annoying nitpick, but, since bellow you explain each transition in the order:
`..., center, outer, wipe, any, random`
these should also be in that exact order: `... | center | outer | wipe | any | random`